backend/bitboxbase/updater/updater.go

```diff
@@ -83,7 +108,150 @@ func (updater *Updater) Connect(address string, bitboxBaseID string) error {
 		return errors.New("updater: Unexpected Response Body Bytes")
 	}
 	updater.running = true
-	go listenWebsocket(updater, bitboxBaseID)
+	updater.log.Printf("connecting to base websocket")
+	client, _, err := websocket.DefaultDialer.Dial("ws://"+updater.address+"/ws", nil)
+	if err != nil {
+		updater.log.Printf("Websocket dial failed: %s", err.Error())
+	}
+	err = updater.initializeNoise(client, bitboxBaseID)
+	if err != nil {
```

Comment:
for shortish lines, i like the inlined version:

`if err := ...; err != nil { ... }`

Reply:
Would be nice to have some guidance on when to use it. For example with functions that only return an error, or function calls up to `n` characters in length.